packages/@lwc/integration-not-karma/configs/hydration.mjs

```diff
@@ -0,0 +1,19 @@
+// Use native shadow by default in hydration tests; MUST be set before imports
+process.env.DISABLE_SYNTHETIC ??= 'true';
```

Comment:
There's probably a more idiomatic way of doing this, but the hack works for now. 🤷